prompts/sources.go

```diff
@@ -163,30 +165,64 @@ func getOverlayPrompts(promptForOverlay *bool, overlayLocation, authHeader *stri
 	return groups
 }
 
-func quickstartBaseForm(ctx context.Context, quickstart *Quickstart) (*QuickstartState, error) {
+func sourceBaseForm(ctx context.Context, quickstart *Quickstart) (*QuickstartState, error) {
 	source := &workflow.Source{}
-	var sourceName, fileLocation, authHeader string
+	var sourceName, fileLocation, authHeader, selectedRemoteNamespace string
+
+	recentGenerations, err := remote.GetRecentWorkspaceGenerations(ctx)
```

Comment:
I would put a relatively short context timeout on this, maybe like 30 seconds. 

The most important thing is still that quickstart works for everybody and sells us well to new users.

Reply:
I went for 10 seconds